lib/wallet/http.js

```diff
@@ -787,14 +787,24 @@ class HTTP extends Server {
 
     // Wallet Name States
     this.get('/wallet/:id/name', async (req, res) => {
+      const valid = Validator.fromRequest(req);
+      const own = valid.bool('own', false);
+
       const height = this.wdb.height;
       const network = this.network;
 
       const names = await req.wallet.getNames();
       const items = [];
 
       for (const ns of names)
-        items.push(ns.getJSON(height, network));
+        if (own) {
```

Comment:
Even though it can be omitted, throughout the codebase any multi-line block needs to be in `{}`.